docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -36,6 +36,17 @@ the index must be older than the minimum age of the next phase.
 The minimum age defaults to zero, which causes {ilm-init} to move indices to the next phase
 as soon as all actions in the current phase complete. 
 
+If an index has unallocated shards and the cluster is in the _yellow_ state, 
+the index can still transition from one phase to the next according to its {ilm} policy, 
+but there might be unexpected side effects because {es} can only perform 
+certain clean up tasks on a healthy cluster. 
+For example, if you have a policy that moves shards to warm nodes when 
+the index transitions out of the hot phase, {es} won't delete shards that 
+weren't fully replicated from the hot nodes. 
```

Comment:
This sentence is confusing for me, and I even know what's going on. To me it sounds mutually exclusive with the previous sentence, saying things like "we don't delete shards that haven't moved over" right after saying "there may be unexpected side effects"